SharpPcap/LibPcap/LibPcapSafeNativeMethods.Encoding.cs

```diff
@@ -37,7 +37,13 @@ internal static partial class LibPcapSafeNativeMethods
 
         private static Encoding ConfigureStringEncoding()
         {
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (!
+#if NET6_0_OR_GREATER
+            OperatingSystem.IsWindows()
+#else
+            RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+#endif
+            )
```

Comment:
Not required for linking `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)` is enough